operatorcourier/cli.py

```diff
@@ -43,9 +43,10 @@ def verify(self):
         parser = argparse.ArgumentParser(description='Build and verify an operator bundle to test')
         parser.add_argument('source_dir', help='Path of your directory of yaml files to bundle. Either set this or use the files argument for bundle data.')
         parser.add_argument('--ui_validate_io', help='Validate bundle for operatorhub.io UI', action='store_true')
+        parser.add_argument('--validation-output', dest='validation_output', help='A file to write validation warnings and erros to in JSON format')
```

Comment:
typo: **erros**. Same typo on the commit message.

Reply:
Good catch - addressed.